// Copyright 2013 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Protocol buffer definitions for Drive backend of Syncable FileSystem.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package sync_file_system.drive_backend;

enum FileKind {
  KIND_UNSUPPORTED = 0;
  KIND_FILE = 1;
  KIND_FOLDER = 2;
}

message ServiceMetadata {
  optional int64 largest_change_id = 1;
  optional string sync_root_folder_id = 2;
}
